src/tools/tests/BUILD

```diff
@@ -1,11 +1,11 @@
+load("//third_party/java/arcs/build_defs:arcs_ts_test.bzl", "arcs_ts_test")
 load(
     "//third_party/java/arcs/build_defs:build_defs.bzl",
     "arcs_cc_schema",
     "arcs_kt_gen",
     "arcs_kt_jvm_test_suite",
     "arcs_kt_schema",
     "arcs_proto_plan",
```

Comment:
I think it's somewhat of an overload for folks to look to import thinks outside of `build_defs:build_defs`. Can we import `arcs_ts_test` in `build_defs` to make this streamlined? 

Was this a deliberate decision? If so, what's the reasoning behind it?

Reply:
`arcs_ts_test` isn't used internally (and doesn't work). In the import CL I've excluded it from the Copybara config so that it won't appear. That was the reason I took it out of build_defs.bzl